### PR TITLE
Add OCA eviction filings to frontend

### PR DESCRIPTION
**Pull request by austensen**
This PR finishes up the project of adding OCA (office of court administration) eviction filings data to WOW. First the backend work was done in #642 to extract the necessary data from the private DB maintained by BetaNYC for the address-level version of the [housing court data ](https://github.com/housing-data-coalition/oca). Then those changes were incorporated into the auto-updating NYCDB (see https://github.com/JustFixNYC/nycdb-k8s-loader/pull/115 & https://github.com/JustFixNYC/nycdb-k8s-loader/pull/117). This PR deals with getting the data from those new database tables and updating the UI to incorporate the new data. It follows the same process as for executed evictions on the overview page, and other indicators on the timeline page, with the one difference being that we can't report filings data on buildings with fewer than 11 units so there is a difference between 0 filings and `N/A` and this is noted in the timeline graph. 

Overview tab:
image

Portfolio tab:
image -> image

Timeline tab:
image

[sc-9478]

### CODE REVIEW

File: client/src/components/PropertiesList.tsx [hunk 1]
```diff
@@ -411,18 +411,29 @@ const TableOfData = React.memo(
...
           },
           {
-            Header: i18n._(t`Evictions`),
+            Header: i18n._(t`Evictions Since 2017`),
             columns: [
               {
                 Header: (
                   <>
-                    <Trans>Since 2017</Trans>
+                    <Trans>Filied</Trans>
```
Comment (shakao) [inline client/src/components/PropertiesList.tsx:419]: small: "filed"

File: client/src/components/PropertiesList.tsx [hunk 2]
```diff
@@ -411,18 +411,29 @@ const TableOfData = React.memo(
...
+                    <ArrowIcon />
+                  </>
+                ),
+                accessor: (d) => d.evictionfilings || null,
+                id: "evictionsfilings",
+                width: getWidthFromLabel(i18n._(t`Filied`)),
```
Comment (shakao) [inline client/src/components/PropertiesList.tsx:425]: small: "filed"

File: sql/create_bldgs_table.sql [hunk 3]
```diff
@@ -89,7 +89,7 @@ select distinct on (registrations.bbl)
   firstdeeds.docamount as lastsaleamount,
   case
     when coalesce(pluto.unitsres, 0) < 11 then NULL
-    else coalesce(oca.eviction_filings_since_2017, 0)
```
Comment (shakao) [inline sql/create_bldgs_table.sql:92]: can you explain this change?

Comment (austensen) [reply sql/create_bldgs_table.sql:92]: properties will only show up in the OCA tables if there are eviction filings, so if the value is missing we can call that zero, except in cases of buildings with < 11 units where we are suppressing data.

Comment (shakao) [reply sql/create_bldgs_table.sql:92]: oh sorry i meant why it moved from being `oca.eviction_filings_since_2017` to just `eviction_filings_since_2017` ?

**Review by shakao — APPROVED** [review]
looks awesome!! 🎉

File: sql/create_bldgs_table.sql (hunk 3 above)
Comment (shakao) [reply sql/create_bldgs_table.sql:92]: ah okay, that makes sense thank you!! that happened here, yes? https://github.com/JustFixNYC/nycdb-k8s-loader/pull/117

Comment (austensen) [reply sql/create_bldgs_table.sql:92]: yes that's right!

**Pull request merged**